tests/nlu/featurizers/test_lm_featurizer.py

```diff
@@ -624,84 +496,76 @@ def test_attention_mask(
                 "rock & roll",
                 "here is the sentence I want embeddings for.",
             ],
+            [[1, 2], [1], [1, 1], [1, 1, 1], [1, 1, 1], [1, 1, 1, 2, 1, 1, 3, 1]],
+        ),
+        (
+            "bert",
+            "bert-base-uncased",
             [
-                ["Good", "even", "ing"],
-                ["hello"],
-                ["you", "re"],
-                ["r", "n", "b"],
-                ["rock", "&", "roll"],
-                [
-                    "here",
-                    "is",
-                    "the",
-                    "sent",
-                    "ence",
-                    "I",
-                    "want",
-                    "embed",
-                    "d",
-                    "ings",
-                    "for",
-                ],
-            ],
-            [
-                [(0, 4), (5, 9), (9, 12)],
-                [(0, 5)],
-                [(0, 3), (4, 6)],
-                [(0, 1), (3, 4), (6, 7)],
-                [(0, 4), (5, 6), (7, 11)],
-                [
-                    (0, 4),
-                    (5, 7),
-                    (8, 11),
-                    (12, 16),
-                    (16, 20),
-                    (21, 22),
-                    (23, 27),
-                    (28, 33),
-                    (33, 34),
-                    (34, 38),
-                    (39, 42),
-                ],
+                "Good evening.",
+                "you're",
+                "r. n. b.",
+                "rock & roll",
+                "here is the sentence I want embeddings for.",
             ],
+            [[1, 1], [1, 1], [1, 1, 1], [1, 1, 1], [1, 1, 1, 1, 1, 1, 4, 1]],
         ),
     ],
 )
-@pytest.mark.skip_on_windows
 def test_lm_featurizer_edge_cases(
-    model_name, model_weights, texts, expected_tokens, expected_indices
+    model_name: Text,
+    model_weights: Text,
+    texts: List[Text],
+    expected_number_of_sub_tokens: List[List[float]],
 ):
-
     if model_weights is None:
         model_weights_config = {}
+        if model_name == "bert" and bool(os.environ.get("CI")):
+            pytest.skip(
+                "Reason: this model is too large, loading it results in"
+                "crashing of GH action workers."
+            )
     else:
         model_weights_config = {"model_weights": model_weights}
-    transformers_config = {**{"model_name": model_name}, **model_weights_config}
+    transformers_config = {
+        **{"model_name": model_name},
+        **model_weights_config,
+        "cache_dir": HF_TEST_CACHE_DIR,
+    }
```

Comment:
Should we move this logic into a separate function since it's reused across multiple tests?